src/main/java/racingcar/RacingMatch.java

```diff
@@ -0,0 +1,45 @@
+package racingcar;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class RacingMatch {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+        String strMatchCars = scanner.nextLine();
+
+        System.out.println("시도할 횟수는 몇 회인가요?");
+        int attemptCount = scanner.nextInt();
+
+        String[] participantCarsName = strMatchCars.split(",");
+
+        ArrayList<Car> cars = new ArrayList<>();
+
+        for (String s : participantCarsName) {
+            cars.add(new Car(s));
+        }
+
+        System.out.println("실행결과");
+        for (int i = 0; i < attemptCount; i++) {
+            cars.forEach(Car::move);
+            System.out.println();
+        }
+
+        List<Car> winners = cars.stream()
+                            .filter(i -> i.getPosition() == cars.stream().max(Comparator.comparingInt(Car::getPosition)).get().getPosition())
+                            .collect(Collectors.toList())
+                            ;
+
+        System.out.print("최종우승자: ");
+        for (int i=0;i<winners.size();i++) {
+            if (i > 0) {
+                System.out.print(", ");
+            }
+            System.out.print(winners.get(i).getName());
+        }
```

Comment:
```suggestion
                String winnerNames = cars.stream()
                .filter(i -> i.getPosition() == cars.stream().max(Comparator.comparingInt(Car::getPosition)).get().getPosition())
                .map(Car::getName)
                .collect(Collectors.joining(", "));
        System.out.printf("최종우승자: %s", winnerNames);
```

중요한 부분은 아니지만 동일한 로직을 `Collectors.joining` 을 이용한다면 간단하게 변경 가능할 것 같아요!!
참고만해주세요! 😆